tgui/src/interfaces/cargo_express.ract

```diff
@@ -21,6 +21,13 @@
 
 {{#if !data.locked }}
   <ui-display title='Express Cargo Console'>
+
+    <ui-section label='Landing Location:'>
```

Comment:
Remove colon, this comes out "Landing Location::".

Reply:
how the fuk did i miss this